internal/server/handlers.go

```diff
@@ -136,18 +136,18 @@ func CreateClusterRole(c echo.Context) error {
 		return c.JSON(http.StatusBadRequest, ErrorRes{err.Error()})
 	}
 
-	type Response struct {
-		Ok bool `json:"ok"`
-	}
-
-	ac.Kubeclient.RbacV1().ClusterRoles().Create(c.Request().Context(), &rbacv1.ClusterRole{
+	_, err := ac.Kubeclient.RbacV1().ClusterRoles().Create(c.Request().Context(), &rbacv1.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: r.RoleName,
 		},
 		Rules: r.Rules,
 	}, metav1.CreateOptions{})
 
-	return c.JSON(http.StatusOK, Response{Ok: true})
+	if err != nil {
+		panic(err.Error())
```

Comment:
why not log.Fatalf?

Reply:
Uniformity. Panic has been used in the rest of the file so im handling the error in the say way, we can decide to switch all panic to fatal